### PR TITLE
Add MCE publish jobs for release-4.14/MCE 2.4

### PR DESCRIPTION
**Pull request by gurnben**
## Summary

This PR adds/enables jobs to publish the release-4.14 branch of the openshift/hypershift repository to the MCE 2.4 pipeline.

### CODE REVIEW

**Comment by gurnben** [comment]
/cc @csrwng 
/cc @bryan-cox 
/cc @sjenning 
/cc @enxebre 

FYI folks - this should enable auto-publish to MCE 2.4 just like we did for MCE 2.3!

**Comment by gurnben** [comment]
Ah - I missed the `make jobs` call - fixing now.

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__mce.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    tag: base
+  openshift_release_golang-1.19:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: stolostron
+    tag: go1.18-linux
```
Comment (bryan-cox) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__mce.yaml:14]: Is there a `go1.19` version of this?

Comment (gurnben) [reply ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__mce.yaml:14]: Yes!  Let me update!

**Review by bryan-cox — APPROVED** [review]
/lgtm

**Comment by csrwng** [comment]
/approve
Thx!

**Pull request merged**